public/index.html

```diff
@@ -7,6 +7,10 @@
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="An app built with every body and heart in mind" />
     <meta name="Angela Battillo" />
+    <meta name="Marceline Ball" />
+    <meta name="Riley Martin" />
+    <meta name="Aiden McGuire" />
+    <meta name="Austin Andrade" />
```

Comment:
Adding author names is great. Looking into it, I think the syntax might be more like this: <meta name="author" content="John Doe">

Reply:
Hey marceline, the syntax You're referencing isn't visible here.

Reply:
Sorry about that! I edited the comment so it is now visible